spot_tools_ros/meshes/spot.mtl

```diff
@@ -0,0 +1,19 @@
+newmtl BlackAbs
+Ns 5.000000
+Ka 1.000000 1.000000 1.000000
+Kd 0.100000 0.100000 0.100000
+Ks 0.500000 0.500000 0.500000
+Ke 0.000000 0.000000 0.000000
+Ni 1.450000
+d 1.000000
+illum 2
+
+newmtl wrap
+Ns 50.000000
+Ka 1.000000 1.000000 1.000000
+Kd 0.878431 0.666666 0.227450
```

Comment:
@nathanhhughes I'm 99% sure the way to recolor spot is to 1) tell the .obj files load `spot_different_color.mtl` instead of this material, and then 2) change Kd to be a different color (Kd is the diffuse color of the material)